### PR TITLE
update manage subscribers page v2.3.x

### LINKED ISSUE #571: Update the Managing Subscribers page
How do I mass unsubscribe customers? I have a list of thousands, and was wondering the best way to do this.

### PR DESCRIPTION
**Pull request by chiranjeevi-cj**
## Purpose of this pull request

This pull request (PR) updates the manage subscriber page.

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/v2.3/marketing/newsletter-subscribers.html

## Additional information

Fixes #571

### CODE REVIEW

File: src/marketing/newsletter-subscribers.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ If you want to use a third-party service to send newsletters, you can export you
 
 1. On the _Admin_ sidebar, go to **Marketing** > _Communications_ > **Newsletter Subscribers**.
 
-1. Find the subscriber in the grid and select the checkbox in the first column.
+   For a multi-site setup where some customers have subscriptions for more than one site, each subscription is displayed as a line item in the grid.
+
+1. Find the subscriber in the grid and select the checkbox in the first column. **Note** For the bulk unsubscribe select the checkbox of each subscriber that you want to cancel.
```
Comment (leslietilling) [inline src/marketing/newsletter-subscribers.md:15]:
```suggestion
1. Find the subscriber in the grid and select the checkbox in the first column.

   {:.bs-callout-info}
   For a bulk unsubscribe, select the checkbox of each subscriber that you want to cancel.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edit for note within a procedure step.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
@dshevtsov  One of the checks is "stuck" -- do we need an update on the `2.3-production` branch to omit this test?

**Comment by dshevtsov** [comment]
@leslietilling, we need an update on the branch to enable this check.
I'll create a PR.

**Comment by dshevtsov** [comment]
#963

**Comment by dshevtsov** [comment]
running tests

**Comment by leslietilling** [comment]
@dshevtsov  Thanks for fixing for this branch :)

**Pull request merged**